tasks/register-runner.yml

```diff
@@ -138,7 +138,7 @@
       --builds-dir '{{ gitlab_runner.builds_dir }}'
       {% endif %}
       {% if gitlab_runner.custom_build_dir_enabled | default(false) %}
-      --custom_build_dir-enabled '{{ gitlab_runner.custom_build_dir_enabled }}'
+      --custom_build_dir-enabled
```

Comment:
Is this working without a true|false argument?

Reply:
Yes. There is no true/false option with the switch.

Reply:
I verified with 'gitlab-runner register --help' lol